rust/signed_doc/src/metadata/doc_type.rs

```diff
@@ -238,6 +238,20 @@ fn map_doc_type(uuid: UuidV4) -> DocType {
     }
 }
 
+/// Maps `DocType` to the deprecated corresponding doc type.
+#[allow(dead_code)]
```

Comment:
Why `allow(dead_code)` it should not been needed here, because its a `pub` function
```suggestion
```